.github/workflows/devbuild.yml

```diff
@@ -40,6 +40,16 @@ jobs:
               sudo curl git build-essential make cmake libc6-dev gcc g++ \
               python3 python3-dev python3-venv
 
+      - name: install and configure gcc-14
```

Comment:
What is the gcc version provided by the ubuntu-24.04 runner?

Reply:
It's 13.3.